### PR TITLE
refactor(#1677): `ProbeMojo` with `cactoos` approach.

### PR DESCRIPTION
**Pull request by volodya-lombrozo**
Apply `cactoos` approach to `probes` method in `ProbeMojo`.

Closes: #1677

### CODE REVIEW

**Comment by volodya-lombrozo** [comment]
@mximp Can you take a look, please?

File: eo-maven-plugin/src/main/java/org/eolang/maven/ProbeMojo.java [hunk 1]
```diff
@@ -222,9 +230,9 @@ private Collection<String> probes(final Path file)
      * @todo #1395:30min Need to add the logic of "hasReservedChars" method to
      *  add-probes.xsl". After that, the method in this class need to be removed.
      */
-    private static boolean hasReservedChars(final String str) {
+    private static boolean notContainsReservedChars(final String str) {
```
Comment (mximp) [inline eo-maven-plugin/src/main/java/org/eolang/maven/ProbeMojo.java:233]: @volodya-lombrozo Clearer name is `missesReservedChars`.

Comment (volodya-lombrozo) [reply eo-maven-plugin/src/main/java/org/eolang/maven/ProbeMojo.java]: @mximp Done

File: eo-maven-plugin/src/main/java/org/eolang/maven/ProbeMojo.java [hunk 2]
```diff
@@ -177,41 +178,48 @@ public void exec() throws IOException {
...
+    }
+
+    /**
+     * Trim Q prefix.
+     * Q.a.b.c -> a.b
+     * a.b.c -> a.b.c
+     * @param obj Full object name
+     * @return Trimmed object name
+     */
+    private static String withoutPrefix(final String obj) {
```
[the remark this answers is not in the document]
Comment (volodya-lombrozo) [reply eo-maven-plugin/src/main/java/org/eolang/maven/ProbeMojo.java]: I've chosen `noPrefix`. `noQ` looks strange, as for me. Done.

**Comment by volodya-lombrozo** [comment]
@mximp All suggestions were fixed. Can you have a look again, please?

**Review by mximp — APPROVED** [review]

**Comment by volodya-lombrozo** [comment]
@yegor256 Can you merge that changes, please?

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**